packages/humanize-distance/src/index.ts

```diff
@@ -1,32 +1,74 @@
+import { IntlShape } from "react-intl";
+
+function roundToOneDecimalPlace(number: number): number {
+  return Math.round(number * 10) / 10;
+}
+
 export function humanizeDistanceStringImperial(
   meters: number,
-  abbreviate?: boolean
+  abbreviate?: boolean,
+  intl?: IntlShape
 ): string {
   const feet = meters * 3.28084;
-  if (feet < 528)
-    return Math.round(feet) + (abbreviate === true ? " ft" : " feet");
-  return Math.round(feet / 528) / 10 + (abbreviate === true ? " mi" : " miles");
+
+  let unit;
+  let value;
+  let unitIfNoIntl;
+
+  if (feet < 528) {
+    unit = "foot";
```

Comment:
Am I right in thinking that `formatMessage` will re-write `foot`?

Reply:
It will also use the correct decimal separator.